src/js/claims-status/components/appeals-v2/Alerts.jsx

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Alert from './Alert';
+import { getAlertContent } from '../../utils/appeals-v2-helpers';
 
-const Alerts = (props) => {
-  if (typeof props.alerts === 'undefined' || props.alerts.length === 0) {
+const Alerts = ({ alerts }) => {
+  if (typeof alerts === 'undefined' || alerts.length === 0) {
     return null;
   }
 
-  const alertsList = props.alerts.map((alert, index) => {
-    const key = `${alert.type}-${index}`;
-    return (<Alert key={key} alert={alert}/>);
-  });
+  const allAlertsContent = alerts.map((alert) => getAlertContent(alert));
+
+  const takeActionAlerts = allAlertsContent.filter((alert) => (alert.displayType === 'take_action'));
+  const infoAlert = allAlertsContent.filter((alert) => (alert.displayType === 'info'));
+
+  const takeActionHeader = (takeActionAlerts.length)
+    ? (<h3>Take Action</h3>)
+    : null;
+
+  const alertsList = takeActionAlerts
+    .concat(infoAlert)
+    .map((alert, index) => {
+      const key = `${alert.type}-${index}`;
+      return (<Alert
+        key={key}
+        title={alert.title}
+        description={alert.description}
+        displayType={alert.displayType}/>);
+    });
 
   return (
-    <ul className="alerts-list">
-      {alertsList}
-    </ul>
+    <div>
```

Comment:
Won't it be nice when we don't have to wrap this in an unnecessary `<div>` just because you add a title?